### ISSUE TITLE
Unclear instructions on https://github.com/RedHatWorkshops/rider-auto-openshift/blob/master/rider-auto-ose-installer/README.md

### ISSUE THREAD

**Issue by alberttwong** [RedHatWorkshops/rider-auto-openshift#14]
says

> Builder Docker containers
> 
> To run these microservices on openshift, you should build the docker images for each module. See the docs for each module for how to do that.

But no command unlike other sections of the README.  Is it implied that I go through every project and execute something?

**Comment by christian-posta** [comment]
Yes. You need to run docker:build to build the images for each module. 

For example, for the file poller module:

`mvn clean install -Pfabric8 -Phawtapp docker:build`

**Comment by alberttwong** [comment]
So there are 6 directories (I'm assuming I don't need to execute in rider-auto-ose-installer or rider-auto-common).
1. rider-auto-backend  
2. rider-auto-file  
3. rider-auto-ws
4. rider-auto-normalizer  
5. rider-auto-rest

Do I execute `mvn clean install -Pfabric8 -Phawtapp docker:build` in each of the 6 directories?

**Comment by christian-posta** [comment]
yes. you need to build the docker images for each module; to do this, you run docker:build in each of the directories (excluding common, that gets included as a dependency to the others). 

then, when you're ready to deploy them to open shift, run the equivalent commands for building the kubernetes json and apply it (i.e., fabric8:json, fabric8:apply)
